Win/Mineways.vcxproj

```diff
@@ -76,6 +76,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <LinkIncremental>false</LinkIncremental>
     <TargetName>$(ProjectName)32</TargetName>
+    <IncludePath>$(VC_IncludePath);$(WindowsSDK_IncludePath);./curl/include;</IncludePath>
```

Comment:
This include line is not needed (I removed it locally to be sure), as
`#include "curl/include/curl.h"`
gets you there just fine.